jsx/PaginationLinks.js

```diff
@@ -1,29 +1,30 @@
 /* exported RPaginationLinks */
 
-let PaginationLinks = React.createClass({
-  mixins: [React.addons.PureRenderMixin],
-  propTypes: {
-    onChangePage: React.PropTypes.func,
-    Total: React.PropTypes.number.isRequired,
-  },
-  getDefaultProps: function() {
-    return {
-      RowsPerPage: 10,
-      Active: 1,
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+class PaginationLinks extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+
     };
-  },
-  changePage: function(i) {
-    let that = this;
+    this.changePage = this.changePage.bind(this);
+  }
+
+  changePage(i) {
     return function(evt) {
       // Don't jump to the top of the page
       evt.preventDefault();
 
-      if (that.props.onChangePage) {
-        that.props.onChangePage(i);
+      if (this.props.onChangePage) {
+        this.props.onChangePage(i);
       }
-    };
-  },
-  render: function() {
+    }.bind(this);
```

Comment:
i'm always unsure about this but if we bind this above in the constructor, do we need to bind it again at the end of the function? looks redundant to me but not sure

Reply:
it looks like this is binding to the returned anon function, not `changePage`, so this usage is :ok_hand: